docs/go/how-to-develop-a-worker-program-in-go.md

```diff
@@ -85,6 +89,8 @@ import SharedWorkersTaskQueueRegistrationNote from '../reminders/note-workers-ta
 
 ### Register multiple Types
 
+Use `RegisterActivity()` for an Activity struct to register all of its exported methods.
```

Comment:
I would move this either above or below the "multiple type" section - and call it out using its own title like:

```
### Registering Activity `structs`

Per [Activity Definition](/docs/how-to-develop-an-activity-definition-in-go) best practices, you may have an Activity struct that has multiple methods and fields.
When an Activity is registered as a struct, that Worker has access to all exported methods and fields.
```

May want to double check on whether the Worker has access to just the exported methods and fields or everything - because I can't recall.

Reply:
Updated.